packages/integrations/markdoc/README.md

```diff
@@ -143,28 +143,26 @@ Use tags like this fancy "aside" to add some *flair* to your docs.
 
 #### Render Markdoc nodes / HTML elements as Astro components
 
-You may also want to map standard HTML elements like headings and paragraphs to components. For this, you can configure a custom [Markdoc node][markdoc-nodes]. This example overrides Markdoc's `heading` node to render a `Heading` component, passing the built-in `level` attribute as a prop:
+You may also want to map standard HTML elements like headings and paragraphs to components. For this, you can configure a custom [Markdoc node][markdoc-nodes]. This example overrides Markdoc's `heading` node to render a `Heading` component, and passes through Markdoc's default attributes for headings via the `Markdoc` object. This is exposed by the `@astrojs/markdoc/config` module and contains all properties from [the `@markdoc/markdoc` package](https://github.com/markdoc/markdoc).
```

Comment:
```suggestion
You may also want to map standard HTML elements like headings and paragraphs to components. For this, you can configure a custom [Markdoc node][markdoc-nodes]. This example overrides Markdoc's `heading` node to render a `Heading` component, and passes through [Markdoc's default attributes for headings](https://markdoc.dev/docs/attributes).
```
Just a suggestion for a more concise structure that I think would be enough for this explanation! I think a link from their docs (pick an appropriate one) is maybe more helpful than the Github link, and lets you pick something more catered to this example.

Reply:
Thanks for that! This section might be doing double duty, trying to tell you that you _can_ access the `Markdoc` global from Astro directly. This has a lot more utility than getting `nodes`, though that's the most common use case I suspect. If this is enough, I think it's good 👍